aiidalab_eln/__init__.py

```diff
@@ -0,0 +1,9 @@
+def get_eln_connector(eln_type: str = "cheminfopy"):
+    if eln_type == "cheminfo":
+        from .eln.cheminfopy import CheminfoElnConnector
```

Comment:
i would maybe call this module simply `cheminfo` as `cheminfopy` is only the Python library we use for making the request a bit easier.

Reply:
agreed.